packages/devtools/lib/src/memory/memory_plotly.dart

```diff
@@ -2,10 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import '../ui/fake_flutter/dart_ui/dart_ui.dart';
+import '../ui/flutter_html_shim.dart';
 import '../ui/plotly.dart';
+import '../ui/theme.dart';
 
 import 'memory_chart.dart';
 
+const ThemedColor chartBackground =
+    ThemedColor(Colors.white, Color.fromRGBO(30, 30, 30, 1));
```

Comment:
can you use the same background color that Devon used for the background of the sidebar in the debugger?

Reply:
Yep didn't notice the values had changed updated.